test/conftest.py

```diff
@@ -22,12 +24,38 @@ def server(app):
 
 
 @pytest.fixture
-def generated_workspace():
+def handled_user():
+    """
+    Create a user, and yield that user.
+
+    On teardown, deletes the user.
+    """
+    user = set_user_cookie(
+        register_user(
+            UserInfo(
+                family_name="test",
+                given_name="test",
+                name="test test",
+                picture="",
+                email="test@test.test",
+                sub=uuid4().hex,
+            )
+        )
+    )
+
+    yield user
+
+    # TODO: Once the function exists, use `delete_user` here instead
+    user_collection().delete(asdict(user))
```

Comment:
Will the `delete_user()` function really be this simple? If yes, then I'd say go ahead and refactor it now.

Reply:
I'm not sure, there may be other things to consider when creating that function. Since this anti-pattern only exists here in the tests, I think it's okay to leave it until it's addressed on its own.